scripts/logger-util.js

```diff
@@ -0,0 +1,24 @@
+const RESET = "\x1b[0m";
+const BOLD = "\x1b[1m";
+const CYAN_BOLD = `${BOLD}\x1b[36m%s${RESET}`;
+const RED = `\x1b[31m%s${RESET}`;
+const YELLOW = `\x1b[33m%s${RESET}`;
+
+module.exports = {
+
+  header: str => {
+    console.log(CYAN_BOLD, str);
+  },
+
+  debug: str => {
+    console.log(str);
+  },
+
+  info: str => {
+    console.log(YELLOW, str);
+  },
+
+  error: str => {
+    console.log(RED, str);
```

Comment:
Is there any advantage of using console.log over console.error ?

Reply:
In this case I think the main advantage would be that we could always turn off the red, or change it to a different color. If we go with console.error it forces the color output to always be red.